software/generic/test/MathTest.cpp

```diff
@@ -55,3 +55,55 @@ TEST(Math, shouldGetCardinalCordinates) {
   ASSERT_EQ(angle, 223);
   ASSERT_EQ(radius, 40);
 }
+
+TEST(Math, shouldMirrorIndiciesOdd) {
+  uint8_t led_index;
+  uint8_t led_count;
+
+  led_index = 0;
```

Comment:
Optional: do you want to make these data driven tests?

Reply:
Sure, do you have an example of one I can follow?

Reply:
Or do I just write the loop myself?

Reply:
I don't have a good example of one - you just write the loop yourself, and use expect. It's a good idea to stream useful error messages to expect, too.

Reply:
Done.